camera.py

```diff
@@ -5,13 +5,15 @@
 import subprocess
 
 import cv2 as cv
+useISP = False
 
 class Camera():
     def __init__(self):
         self.camera_device = None
         self.color_conversion_code = None
         self.api_preference = None
         self.video_capture = cv.VideoCapture()
+        self.useISP: bool = False
```

Comment:
```suggestion
        self.use_isp: bool = False
```
Also add this to the `CameraVM016` class as it is not applicable to the USB cameras.